pkg/operator/config.go

```diff
@@ -82,6 +99,19 @@ func getProviderControllerFromImages(platform configv1.PlatformType, images Imag
 	}
 }
 
+// This function returns images required to bring up the Baremetal Pod.
+func getBaremetalControllers(images Images) (BaremetalControllers, error) {
```

Comment:
This never returns an error, let's remove the error return.

Reply:
Also, let's rename to newBaremetalControllers as that's more descriptive since it's basically a constructor/factory function.

Reply:
```
func newBaremetalControllers(images Images, usingBareMetal bool) *BaremetalControllers
```